.github/component-owners.yml

```diff
@@ -28,9 +28,10 @@ components:
   source/devices/AM335x/linux: &sitara_legacy_list
     - jeevantelukula
     - jmenti
-  source/devices/AM437x/linux: *sitara_legacy_list
```

Comment:
You and @jeevantelukula are already added as reviwers for AM437X and AM65X via *sitara_legacy_list
Any reasons for creating this PR?

Reply:
Actually I had misunderstood the yaml syntax, well revert, thanks